src/pages/settings/Wallet/ExpensifyCardPage.tsx

```diff
@@ -204,39 +249,42 @@ function ExpensifyCardPage({
                                             icon={Expensicons.Flag}
                                             shouldShowRightIcon
-                                            onPress={() => Navigation.navigate(ROUTES.SETTINGS_REPORT_FRAUD.getRoute(domain))}
-                                        />
-                                    </>
-                                )}
-                                {physicalCard?.state === CONST.EXPENSIFY_CARD.STATE.OPEN && (
-                                    <>
-                                        <MenuItemWithTopDescription
-                                            description={translate('cardPage.physicalCardNumber')}
-                                            title={CardUtils.maskCard(physicalCard?.lastFourPAN)}
-                                            interactive={false}
-                                            titleStyle={styles.walletCardNumber}
-                                        />
-                                        <MenuItem
-                                            title={translate('reportCardLostOrDamaged.report')}
-                                            icon={Expensicons.Flag}
-                                            shouldShowRightIcon
-                                            onPress={() => Navigation.navigate(ROUTES.SETTINGS_WALLET_REPORT_CARD_LOST_OR_DAMAGED.getRoute(domain))}
+                                            onPress={() => Navigation.navigate(ROUTES.SETTINGS_REPORT_FRAUD.getRoute(domain, cardId))}
                                         />
                                     </>
+                                ))}
+                                {physicalCards.map(
+                                    (card) =>
+                                        card.state === CONST.EXPENSIFY_CARD.STATE.OPEN && (
+                                            <>
+                                                <MenuItemWithTopDescription
+                                                    description={translate('cardPage.physicalCardNumber')}
+                                                    title={CardUtils.maskCard(card?.lastFourPAN)}
+                                                    interactive={false}
+                                                    titleStyle={styles.walletCardNumber}
+                                                />
+                                                <MenuItem
+                                                    title={translate('reportCardLostOrDamaged.report')}
+                                                    icon={Expensicons.Flag}
+                                                    shouldShowRightIcon
+                                                    onPress={() => Navigation.navigate(ROUTES.SETTINGS_WALLET_REPORT_CARD_LOST_OR_DAMAGED.getRoute(domain, cardId))}
+                                                />
+                                            </>
```

Comment:
in such cases I personally prefer to do the longer version of arrow: `() => { ... }` so that I can do an early return, instead of `&&`.

I find its easier to reason what gets returned if the code looks something like this:
```ts
{
  [...]
  if (!SOME_CONSTS) {
    return null
  }

  return <>JSX....</>
}

```